content/en/docs/concepts/workloads/pods/pod-overview.md

```diff
@@ -74,9 +74,9 @@ Here are some examples of workload resources that manage one or more Pods:
 
 ## Pod templates
 
-Generally: controllers for workload resources use a Pod template (that you provide) to create the Pods for that workload. The controller is responsible for managing those Pods on your behalf.
+Generally: controllers for {{< glossary_tooltip text="workload" term_id="workload" >}} resources use a Pod template (that you provide) to create the Pods for that workload. The controller is responsible for managing those Pods on your behalf.
```

Comment:
Line 77, could the sentence be simplified to something like:

Controllers, for {{< glossary_tooltip text="workload" term_id="workload" >}} resources, create Pods from a Pod template and manage those Pods on your behalf.